tests/test_pow2.py

```diff
@@ -25,5 +25,11 @@ def test_interpolated(taso_256_lpf, output_30hz):
 
 
 def test_32hz(input_32hz, output_30hz):
-    resampled = resample_to_30hz(input_32hz, 32)
+    resampled = resample_to_30hz(input_32hz, sample_rate=32)
     assert np.abs(resampled - output_30hz).max() == pytest.approx(0)
+
+
+def test_32hz_interpolate(input_32hz, output_30hz):
+    resampled = resample_to_30hz(input_32hz, sample_rate=32, use_mne_filter=True)
+    assert np.abs(resampled - output_30hz).max() < 1
+    assert resampled.mean() == pytest.approx(-5.8447899368136745e-05)
```

Comment:
I don't want to be nitpicking, but the mean of the difference would be a stronger test
`np.abs(resampled - output_30hz).mean() < xxx`